testkit/javadsl/src/main/scala/com/lightbend/lagom/javadsl/testkit/ServiceTest.scala

```diff
@@ -68,6 +68,8 @@ object ServiceTest {
   private val KafkaBrokerModule = "com.lightbend.lagom.internal.javadsl.broker.kafka.KafkaBrokerModule"
   private val KafkaClientModule = "com.lightbend.lagom.javadsl.broker.kafka.KafkaClientModule"
 
+  private val LagomTestConfigResource: String = "lagom-test-embedded-cassandra.yaml"
+
```

Comment:
Variable casing looks odd. I see previous variables used similar convention but it's a different type of item they represent.